src/components/Home/Home.tsx

```diff
@@ -36,7 +36,6 @@ class Home extends Component {
 				<img
 					src={process.env.PUBLIC_URL + '/logo.png'}
 					alt="logo"
-					className={styles.logoImage}
```

Comment:
Did this style even exist for this component? I know it's on App but not Home?

Reply:
Nope. I was wondering that as well, because typescript was ree'ing at me as soon as I switched that stuff over. It's not in the styles at all.